cl/recap/management/commands/pacer_iquery_scraper.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+from __future__ import print_function
+
+import json
+import re
+
+import requests
+from django.conf import settings
+
+from cl.celery import app
+from cl.lib.command_utils import VerboseCommand
+from cl.lib.pacer import map_cl_to_pacer_id
+from cl.lib.pacer_session import get_or_cache_pacer_cookies
+from cl.recap.mergers import (
+    update_docket_metadata,
+    add_bankruptcy_data_to_docket,
+)
+from cl.search.models import Docket
+from cl.search.tasks import add_items_to_solr
+from juriscraper.pacer import CaseQuery, PacerSession
+
+
+def get_dockets():
+    visits = requests.get(
+        settings.MATOMO_URL,
+        timeout=1,
+        params={
+            "idsite": settings.MATOMO_SITE_ID,
+            "module": "API",
+            "method": "Live.getLastVisitsDetails",
+            "period": "week",
+            "format": "json",
+            "date": "today",
+        },
+    )
+    visitsjson = json.loads(visits.text)
+    for item in visitsjson[:10]:
+        print(item["actionDetails"][0]["url"])
+    urllist = [
+        item["actionDetails"][0]["url"]
+        if "url" in item["actionDetails"][0]
+        else ""
+        for item in visitsjson
+    ]
+    urllistasstring = "|".join(urllist)
+    return set(re.findall("/docket/([0-9]+)/", urllistasstring))
+
+
+@app.task()
```

Comment:
This still needs to go in the tasks.py file to be found (I think!). Even if not, it's probably best there.

Reply:
I can move the update iquery function back there. It's not getting call

Reply:
You're right, it shouldn't be in alerts/tasts.py, but it could work nicely in scrapers/tasks.py